docs/api-routes/api-middlewares.md

```diff
@@ -68,6 +68,30 @@ export const config = {
 }
 ```
 
+`bodyLimit` is automatically enabled, warning when an API routes' response is over 4MB. If you'd like to disable this warning, you can set this to false.
+
+A use case for disabling the `bodyLimit` warning is if you are deploying outside of the Vercel ecosystem, i/e AWS or self hosting.
+
+```js
+export const config = {
+  api: {
+    bodyLimit: false,
```

Comment:
Should we call this `responseLimit` instead to prevent confusion with the request body?

Reply:
Works for me!